docs/content/cache.md

```diff
@@ -133,7 +133,7 @@ to the cloud provider without interrupting the reading (small blip can happen th
 
 Files are uploaded in sequence and only one file is uploaded at a time.
 Uploads will be stored in a queue and be processed based on the order they were added.
-The queue and the temporary storage is persistent across restarts and even purges of the cache.
+The queue and the temporary storage is persistent across restarts and can purge the cache on startup.
```

Comment:
I'm still not sure if I get the full meaning of this sentence.
Does the sentence express that the queue and temporary storage can be cleared at the start by using `--cache-db-purge`?
If this is the case then the following wording would make it clearer to me:
> ... persistent across restarts, but can be cleared on startup with \`--cache-db-purge\`.